cfme/tests/cloud/test_providers.py

```diff
@@ -727,42 +871,74 @@ def test_domain_id_validation(request, provider):
 
 @pytest.mark.manual
 @pytest.mark.tier(1)
-def test_upload_azure_image_to_azure():
+def test_sdn_nsg_arrays_refresh_azure():
     """
-    Upload image copied from RHCF3-11271 to azure using powershell
-    similiar the script in the setup section.
+    https://bugzilla.redhat.com/show_bug.cgi?id=1520196
```

Comment:
Please use `@pytest.mark.meta(automates=[BZ(1520196)])` to indicate test coverage.

Reply:
hmmm, as it's a manual test, I don't think it's correct.
I'm adding it like this
```
    Bugzilla:
        1520196
```
as I see it in other _manual_ tests as well